sklearn/feature_selection/from_model.py

```diff
@@ -108,6 +108,9 @@ class SelectFromModel(BaseEstimator, SelectorMixin, MetaEstimatorMixin):
         Otherwise train the model using ``fit`` and then ``transform`` to do
         feature selection.
 
+    max_features : int, between 0 and number of features, optional.
+        Select at most this many features that score above the threshold.
```

Comment:
Could you please add a note that to use only `max_features`, and no threshold, `threshold=-np.inf` can be used. But perhaps we should allow users to disable threshold with a string `'-inf'`? (I'd rather not `'none'` which will get confused with the current None, meaning automatic threshold determination. Although we could in turn consider deprecating the use of `threshold=None` and renaming it to `threshold='auto'`.)